src/v3/components/card_header/index.styles.ts

```diff
@@ -0,0 +1,13 @@
+import { Box } from '@mui/material';
+import { styled } from '@mui/system';
+import { CardHeaderSizeType } from './index.types';
+
+export const StyledContentBox = styled(Box)<{ color: string; size: CardHeaderSizeType }>((props) => ({
```

Comment:
@Logan08691: please fix this code in order for these props to not end up in the DOM. Probably, they will cause too much noises in the dev console.